deploy.sh

```diff
@@ -3,14 +3,11 @@
 set -u
 set -o pipefail
 
-readonly __SCRIPT_NAME__="${0##*/}"
-readonly __SEE_HELP_MESSAGE__="See '${__SCRIPT_NAME__} --help' for more information."
-
-GH_REMOTE='origin'
-SOURCE_BRANCH='main'
-DEPLOY_BRANCH='gh-pages'
-WEBSITE_DIR_PATH='dist'
-RETURN_PATH='..'
+readonly GH_REMOTE="origin"
```

Comment:
Hmm. I had found it useful to specify this via the CLI - it's how I QA'd previous changes to the deploy script for example.